test/hotspot/jtreg/vmTestbase/nsk/jvmti/scenarios/hotswap/HS201/hs201t002/hs201t002.cpp

```diff
@@ -377,7 +379,7 @@ callbackExceptionCatch(jvmtiEnv *jvmti_env, JNIEnv* jni_env, jthread thread,
 
 /* ============================================================================= */
 
-int readNewBytecode(jvmtiEnv* jvmti) {
+int readNewBytecode(jvmtiEnv* jvmti, jint &newClassSize, unsigned char* &newClassBytes) {
```

Comment:
The function returns loaded data instead of using global variable for correct resource management